faker/providers/python/__init__.py

```diff
@@ -320,7 +320,10 @@ def pydecimal(
         elif max_value is not None and max_value <= 0:
             sign = "-"
         else:
-            sign = "+" if positive else self.random_element(("+", "-"))
+            if positive == None:
```

Comment:
Just a stylistical note: could we use `is` to compare to `None`?

```suggestion
            if positive is None:
```

Reply:
added the changes in [bf1055a](https://github.com/joke2k/faker/pull/2080/commits/bf1055ace76df99e7a403275cf997c165ed325a8)